### ISSUE TITLE
[WebToolsE2E][Aspire] After successfully publishing an Aspire project, the Console log does not have any logs in the 'Aspire Dashboard'

### ISSUE THREAD

**Issue by Rita003** [dotnet/aspire#5148]
**INSTALL STEPS**
1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 8.0.303
3. Install Aspire 8.1.0
4. Install latest released azd v1.9.5
5. Install git v2.46.0

**REPRO STEPS**
1. Open CMD, run following command to create an Aspire Starter App project and deploy it
```
dotnet new aspire-starter -o AspireStarterApp
cd AspireStarterApp/AspireStarterApp.AppHost
azd init
azd up
```
2. After successful deployment, open the deployed "Aspire Dashboard" URL
![Screenshot 2024-08-02 142740](https://github.com/user-attachments/assets/b53944a8-fff1-405b-bbfb-f09e2c092788)
3. Click the "Console" option on the Aspire dashboard left, and select the apiservice/webfrontend resource.

**Note:**
1. This issue also repro on azd v1.9.4 & azd v1.9.3

**ACTUAL**
Console log does not have any log information
![Screenshot 2024-08-02 143347](https://github.com/user-attachments/assets/92b16f4c-d008-4455-a4a3-f7c508db4e27)
![Screenshot 2024-08-02 143435](https://github.com/user-attachments/assets/369ea67d-cd5d-46d0-aabd-6e8d6b8c3250)

**EXPECTED**
Console log has some log information

**Comment by adamint** [comment]
@snehapar9

**Comment by adamint** [comment]
@snehapar9

**Comment by Rita003** [comment]
Verify this issue fixed on the latest Aspire 8.1.1-preview.1.24421.2 + azd v1.9.6, close it now.
